scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -42,16 +38,7 @@ function readConfig(): FrameworkOptions {
   return JSON.parse(data);
 }
 
-function logIncludes(msg: string) {
-  return (data: unknown) => {
-    if (typeof data !== "string" && !Buffer.isBuffer(data)) {
-      throw new Error(`data is not a string or buffer (${typeof data})`);
-    }
-    return data.includes(msg);
-  };
-}
-
-describe("database and firestore emulator function triggers", () => {
+describe("function triggers", () => {
```

Comment:
These changes look very large, but they are really just reorganizing these tests to share a single beforeAll hook & to run as part of the same emulator run.